server/csv-renderers/statistics-flow-operators.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var ensureObject     = require('es5-ext/object/valid-object')
+  , deferred         = require('deferred')
+  , modes            = require('../../utils/statistics-flow-group-modes')
+  , _                = require('mano').i18n.bind('Statistics operators csv')
+  , usersFullNameMap = require('../lib/user-full-names-map');
+
+module.exports = function (result, config) {
+	ensureObject(config);
+	var data, mode;
+	mode = modes.get(config.mode || 'daily');
+	data = [
+		[
+			mode.labelNoun,
```

Comment:
This doesn't reflect expected format, as described in this comment -> https://github.com/egovernment/eregistrations/issues/1759#issuecomment-275636701

First column was supposed to state step, and then we should list data for all steps, one after another